resources/server/bin/helpers/check-requirements-linux.sh

```diff
@@ -19,6 +28,12 @@ if [ -f "/tmp/vscode-skip-server-requirements-check" ]; then
 	exit 0
 fi
 
+# Default to legacy server if the following file is present.
+if [ -f "$HOME/@@QUALITY@@-use-legacy" ]; then
+    echo "!!! WARNING: Using legacy server !!!"
```

Comment:
maybe say that legacy sever is selected due to the existence of the $HOME/@@QUALITY@@-use-legacy file